orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -117,7 +124,8 @@
 						GKInstance firstPsiModInst = (GKInstance) modifiedResidueInst.getAttributeValue(psiMod);
 						if (phosFlag && firstPsiModInst.getAttributeValue(name).toString().contains("phospho")) {
 							String phosphoName = "phospho-" + infEWASInst.getAttributeValue(name);
-							List<GKInstance> ewasNames = (ArrayList<GKInstance>) infEWASInst.getAttributeValuesList(name);
+							List<String> ewasNames = (ArrayList<String>) infEWASInst.getAttributeValuesList(name);
+							String originalName = ewasNames.get(0);
```

Comment:
This line and the next could be combined since the remove method will return the element that is being removed:

`String originalName = ewasNames.remove(0)`